requirements.txt

```diff
@@ -13,6 +13,7 @@ pytest
 requests-mock
 flake8
 tox
+scipy
```

Comment:
Is this still a requirement? I don't see it imported anywhere

Reply:
Ah, I forgot to commit this change. Updating now.